etc/matrix/homeserver.yaml.template

```diff
@@ -22,7 +22,7 @@ listeners:
 database:
   name: sqlite3
   args:
-    database: ${DEPLOYMENT_MATRIX_CONTAINER_NAME}_data/homeserver.db
```

Comment:
No @aycherif, you will add the volume for the container and the path will be kept as it is inside it